### PR TITLE
feat: add fdb support to the runner

### PR DESCRIPTION
**Pull request by pboros**
Add fdb benchmarking support for the runner.

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -46,45 +49,96 @@ requestdistribution=${REQUESTDISTRIBUTION}"
...
-	echo "Invalid value in RUNMODE variable. Choose between single and multiple_threads."
-fi
+echo "Using engine ${ENGINE}"
+
+case ${ENGINE} in
+	"tigris")
+		# Check if the client works
+		echo "Checking if tigris client is ok"
+
+		${CLI_PATH}tigris list databases
```
Comment (efirs) [inline run.sh]: You can use `tigris ping` or even `tigris ping --timeout 10s` if you want it to wait for server to come up.
However, timeout has some pending fixes and will be available once release is unblocked.

File: run.sh [hunk 2]
```diff
@@ -46,45 +49,96 @@ requestdistribution=${REQUESTDISTRIBUTION}"
...
+					timeout ${RUNTHREADDURATION} ${BIN_PATH}/go-ycsb run tigris -p tigris.host="$TIGRIS_HOST" -p tigris.port="$TIGRIS_PORT" -p tigris.dbname="$TEST_DB" -P workloads/dynamic -p threadcount=${th}
+					sleep ${RUNTHREADSLEEPINTERVAL}
+				done
+			done
+		else
+			echo "Invalid value in RUNMODE variable. Choose between single and multiple_threads."
+		fi
+	;;
+  "foundationdb")
+  	echo "Using FDB_CLUSTER_FILE ${FDB_CLUSTER_FILE}"
```
Comment (efirs) [inline run.sh]: I think it's better to move this to a function and tigris branch too.

File: run.sh [hunk 3]
```diff
@@ -46,45 +49,96 @@ requestdistribution=${REQUESTDISTRIBUTION}"
...
+
+		echo "Using TIGRIS_HOST ${TIGRIS_HOST}"
+		echo "Using TIGRIS_PORT ${TIGRIS_PORT}"
+		echo "Using TEST_DB ${TEST_DB}"
+		echo "Using LOADTHREADCOUNT ${LOADTHREADCOUNT}"
+
+		if [ "${DROPANDLOAD}" -gt 0 ]
+		then
+			echo "Dropping test database"
+			${CLI_PATH}tigris drop database $TEST_DB
```
Comment (efirs) [inline run.sh]: Heads up, this will be called delete-project in the next release

**Review by efirs — DISMISSED** [review]

File: run.sh (hunk 1 above)
Comment (pboros) [reply run.sh]: Thanks, I will use tigris ping. I just need to bail out if it doesn't work, k8s will take care of the rest.

File: run.sh (hunk 3 above)
Comment (pboros) [reply run.sh]: Thanks, I will change it with that!

File: run.sh (hunk 2 above)
Comment (pboros) [reply run.sh]: Good idea, I did that.

**Review by rbarabas — APPROVED** [review]

File: run.sh [hunk 4]
```diff
@@ -44,47 +47,106 @@ insertproportion=${INSERTPROPORTION}
...
+	else
+		echo "Invalid value in RUNMODE variable. Choose between single and multiple_threads."
+	fi
+}
+
+echo "${WORKLOAD}" > workloads/dynamic
+
+echo "Using engine ${ENGINE}"
+
+case ${ENGINE} in
```
Comment (rbarabas) [inline run.sh:140]: We could escape this as it's user input

**Pull request merged**